docs/playground/ko/TypeScript/Type Primitives/Built-in Utility Types.ts

```diff
@@ -0,0 +1,121 @@
+//// { order: 3, compiler: { strictNullChecks: true } }
+
+// 대부분 코드에서 특정한 타입이 유용하다고 느낄 때,
+// TypeScript에 추가해서
+// 누구나 사용할 수 있게 하여
+// 가용성을 지속해서 의존할 수 있습니다
+
+// Partial<Type>
+
+// 타입을 갖고 모든 프로퍼티를
+// 선택적 프로퍼티로 변환하세요.
+
+interface Sticker {
+  id: number;
+  name: string;
+  createdAt: string;
+  updatedAt: string;
+  submitter: undefined | string;
+}
+
+type StickerUpdateParam = Partial<Sticker>;
+
+// Readonly<Type>
+
+// 객체를 갖고 읽기 전용 프로퍼티를 만드세요.
+
+type StickerFromAPI = Readonly<Sticker>;
+
+// Record<KeysFrom, Type>
+
+// KeysFrom의 프로퍼티 리스트를 사용하는 타입을 만들고
+// Type의 값을 전달하세요.
+
+// List which keys come from:
```

Comment:
```suggestion
// List which keys come from:
```
@yahma25 누락된 부분인 것 같습니다 :)

Reply:
😲 확인 감사해요. 👍  수정하겠습니다~